src/ui/components/pages/otherAccounts.tsx

```diff
@@ -44,10 +44,25 @@ export function OtherAccountsPage() {
   const balancesMoney = Object.fromEntries(
     Object.entries(balances).map(([key, balance]) => [
       key,
-      balance && new Money(balance.available, wavesAsset),
+      typeof balance?.regular !== 'undefined'
+        ? new Money(balance.regular, wavesAsset)
+        : undefined,
     ])
   );
 
+  useEffect(() => {
+    background.updateOtherAccountsBalances();
+
+    const intervalId = setInterval(
```

Comment:
I thought we discussed that we'd better start next request after previous one is done. Am I wrong? If not, then `setInterval` should be replaced with `setTimeout`. And `setTimeout` should be called only after the update is done.